### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by redhatrises**
- Add OpenShift install instructions for CLI and Web Console
- Update some workflows to address some Cx issues
- Depends on #235

### CODE REVIEW

File: docs/deployment/openshift/README.md [hunk 1]
```diff
@@ -1,41 +1,262 @@
...
+- [FalconContainer](../../cluster_resources/container/README.md) custom resource to the cluster with Falcon Container image being mirrored from CrowdStrike container registry to OpenShift ImageStreams (on cluster registry).
+- [FalconNodeSensor](../../cluster_resources/node/README.md) custom resource to the cluster.
+
+You can choose to install the operator and custom resources through the [web console (GUI)](#installing-the-operator-through-the-web-console-gui) or through the [CLI](#installing-the-operator-through-the-cli).
+If you want to automate the deployment of the operator, the CLI method is recommended.
+
+## Prerequisites
+
+- CrowdStrike CWP subscription with Falcon Container
+- If your are installing the CrowdStrike Sensos via the Crowdstrike API, you need to create a new CrowdStrike API key pair with the following permissions:
```
Comment (jsullivan-CS) [inline docs/deployment/openshift/README.md:12]: typo, Sensos/Sensor

File: docs/deployment/openshift/README.md [hunk 2]
```diff
@@ -1,41 +1,262 @@
...
+  or to get the package manifest in yaml form:
+  ```
+  oc get packagemanifests -n openshift-marketplace falcon-operator -o yaml
+  ```
+  Important information from the package manifest output such as the `defaultChannel`, `catalogSource`, `catalogSourceNamespace`, and `currentCSV` are used to create a `Subscription` Kind in a yaml file (next steps) to have OpenShift install the operator from the cluster's marketplace.
+  You can install either the [Community version of the Operator](#installing-the-community-operator-from-the-local-operatorhub) or the official [Red Hat Marketplace certified version of the operator](#installing-the-red-hat-marketplace-operator-from-the-local-operatorhub).
+
+### Installing the Red Hat Marketplace Operator from the Console OperatorHub
+
+- Create a subscription `yaml` file to install the official Red Hat Marketplace certified operator (`redhat-marketplace`). In this example, version 0.5.4 of the community operator will be installed via the `Subscription` Kind:
```
Comment (jsullivan-CS) [inline docs/deployment/openshift/README.md:100]: This should be the RH marketplace example.

File: docs/deployment/openshift/README.md (hunk 1 above)
Comment (redhatrises) [reply docs/deployment/openshift/README.md]: Ha... thanks.

File: docs/cluster_resources/node/README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+```
+
+### FalconNodeSensor Reference Manual
+
+#### Falcon API Settings
+| Spec                                | Description                                                                                                                               |
+| :---------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------- |
+| falcon_api.client_id                | (optional) CrowdStrike API Client ID                                                                                                      |
+| falcon_api.client_secret            | (optional) CrowdStrike API Client Secret                                                                                                  |
+| falcon_api.client_region            | (optional) CrowdStrike cloud region (allowed values: autodiscover, us-1, us-2, eu-1, us-gov-1)                                            |
```
Comment (rkubik-CS) [inline docs/cluster_resources/node/README.md:52]: Should this be `cloud_region`? Similar comment for `docs/cluster_resources/container/README.md`.

Comment (utpk) [reply docs/cluster_resources/node/README.md:52]: Just saw this comment, I've crated #342 for this 🤙

**Pull request merged**